practice-one/src/index.html

```diff
@@ -44,6 +45,64 @@
     </div>
 
   </header>
+  <!-- END HEADER -->
+
+  <!-- START MAIN -->
+  <main class="main-content">
+
+    <div class="container container-main">
+
+      <!-- START SECTION GET-STARTED -->
+      <section class="section-get-started">
+
+        <section class="download">
+
+          <h1 class="download-heading">
+            Download app,<br />
+            save money, make<br />
+            friends!
+          </h1>
+
+          <p class="content download-content">
+            Its simple and its free. Play your part in reducing Carbon Footprint and help
+            Mother Nature to sustain its beauty. So what are you waiting for? Lets ride
+            together
+          </p>
+
+          <div class="download-links">
+            <a href="#" class="btn btn-link btn-download">
+              <svg class="download-icon" xmlns="http://www.w3.org/2000/svg" width="45" height="41" viewBox="0 0 45 41"
```

Comment:
Should create an SVG file after using an image tag to embed SVG into HTML. Let use SVG sprites in this case